### PR TITLE
Update scalameta to 4.4.14

### PR DESCRIPTION
**Pull request by tgodzik**
Fixes https://github.com/scalameta/scalafmt/issues/2441

### CODE REVIEW

File: scalafmt-tests/src/test/resources/scala3/Issues.stat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+<<< extends with scalafmt/issues/2441
+object ValueTypes:
+    case object Private
+        extends ClassificationValue
+        with ConstantText
+>>>
+object ValueTypes:
+  case object Private extends ClassificationValue with ConstantText
```
Comment (kitbellew) [inline scalafmt-tests/src/test/resources/scala3/Issues.stat:9]: missing newline at end of file

**Review by kitbellew — CHANGES_REQUESTED** [review]
context function test is failing...

**Comment by tgodzik** [comment]
> context function test is failing...

I noticed that one :thinking: We actually didn't insert indentation before after ?=>, so that might be the cause. I will take a look later on.

**Comment by kitbellew** [comment]
@tgodzik something with positions again: the function includes a closing parenthesis from the parent apply:
```
TYPE: Term$ContextFunction$TermContextFunctionImpl
SOURCE: err ?=>
        {
            logger.warn(s"Malformed request: $err\n${req.body}")
            BadRequest(jsonError(JsError toJson err)).fuccess
        })
STRUCTURE: Term.ContextFunction(List(Term.Param(Nil, Term.Name("err"), None, None)), Term.Block(List(Term.Apply(Term.Select(Term.Name("logger"), Term.Name("warn")), List(Term.Interpolate(Term.Name("s"), List(Lit.String("Malformed request: "), Lit.String("\\n"), Lit.String("")), List(Term.Name("err"), Term.Select(Term.Name("req"), Term.Name("body")))))), Term.Select(Term.Apply(Term.Name("BadRequest"), List(Term.Apply(Term.Name("jsonError"), List(Term.ApplyInfix(Term.Name("JsError"), Term.Name("toJson"), Nil, List(Term.Name("err"))))))), Term.Name("fuccess")))))
TOKENS: err,∙,?=>,¶,∙,∙,∙,∙,∙,∙,∙,∙,{,¶,∙,∙,∙,∙,∙,∙,∙,∙,∙,∙,∙,∙,logger,.,warn,(,s,",Malformed∙request:∙,$,err,,\n,$,{,req,.,body,},,,",),¶,∙,∙,∙,∙,∙,∙,∙,∙,∙,∙,∙,∙,BadRequest,(,jsonError,(,JsError,∙,toJson,∙,err,),),.,fuccess,¶,∙,∙,∙,∙,∙,∙,∙,∙,},)
```

**Comment by tgodzik** [comment]
We'll need to update to the next version.

**Pull request closed without merging**